### PR TITLE
test(@wallet): test add saved address

### LINKED ISSUE #6761: Automation: Can add an address to Saved addresses

### PR DESCRIPTION
**Pull request by alaibe**
fixes #6761

### CODE REVIEW

File: test/ui-test/testSuites/suite_status/shared/scripts/names.py [hunk 1]
```diff
@@ -162,9 +162,23 @@
 navBarListView_Wallet_navbar_StatusNavBarTabButton = {"checkable": True, "container": mainWindow_navBarListView_ListView, "objectName": "Wallet-navbar", "type": "StatusNavBarTabButton", "visible": True}
 wallet_navbar_wallet_icon_StatusIcon = {"container": navBarListView_Wallet_navbar_StatusNavBarTabButton, "objectName": "wallet-icon", "type": "StatusIcon", "visible": True}
 
-mainWallet_Account_Name = {"container": statusDesktop_mainWindow, "objectName": "accountName", "type": "StatusBaseText", "visible": True}
+# Wallet main view:
 
+mainWallet_Account_Name = {"container": statusDesktop_mainWindow, "objectName": "accountName", "type": "StatusBaseText", "visible": True}
+mainWallet_Saved_Addresses_Button = {"container": statusDesktop_mainWindow, "objectName": "savedAddresseBtn", "type": "StatusButton"}
 mainWallet_Footer_Send_Button = {"container": statusDesktop_mainWindow, "objectName": "walletFooterSendButton", "type": "StatusFlatButton"}
+mainWallet_Add_Account = {"container": statusDesktop_mainWindow, "text": "Add account", "type": "StatusBaseText", "unnamed": 1, "visible": True}
```
Comment (jrainville) [inline test/ui-test/testSuites/suite_status/shared/scripts/names.py:170]: Let's try to avoid unamed objects. Also using `text` is prone to design change.

**Review by jrainville — APPROVED** [review]
Good job. If you want, it would be greatly appreciated if you rebased your PR on top of my cleanup PR. I'm trying to reduce the painful rebases: https://github.com/status-im/status-desktop/pull/6735
Same thing if you work on other test issues.

**Comment by alaibe** [comment]
@jrainville no problem, I will wait til you have this one merge and rebase on top of it

File: test/ui-test/testSuites/suite_status/shared/scripts/names.py (hunk 1 above)
Comment (alaibe) [reply test/ui-test/testSuites/suite_status/shared/scripts/names.py:170]: Not adding this one, just moving, this is legacy

File: ui/app/AppLayouts/Wallet/views/LeftTabView.qml [hunk 2]
```diff
@@ -127,6 +127,7 @@ Rectangle {
         }
 
         StatusButton {
+            objectName: "savedAddresseBtn"
```
Comment (caybro) [inline ui/app/AppLayouts/Wallet/views/LeftTabView.qml:130]: Typo? "savedAddress_e_Btn"

File: ui/app/AppLayouts/Wallet/views/SavedAddressesView.qml [hunk 3]
```diff
@@ -69,6 +70,7 @@ Item {
         StatusListItem {
             id: savedAddress
             title: name
+            objectName: name
```
Comment (caybro) [inline ui/app/AppLayouts/Wallet/views/SavedAddressesView.qml:73]: I suggest using sth like `"savedAddress-%1".arg(name)`

**Review by caybro — COMMENTED** [review]
Some suggestions, otherwise looks good imho

File: ui/app/AppLayouts/Wallet/views/LeftTabView.qml (hunk 2 above)
Comment (alaibe) [reply ui/app/AppLayouts/Wallet/views/LeftTabView.qml:130]: yep, Fixed

File: ui/app/AppLayouts/Wallet/views/SavedAddressesView.qml (hunk 3 above)
Comment (alaibe) [reply ui/app/AppLayouts/Wallet/views/SavedAddressesView.qml:73]: As this is part of a list view, this is then much easer without any prefix as we can reuse the code more easily to find element into a list

**Review by caybro — APPROVED** [review]

**Pull request merged**